pkg/plugins/frontend_plugin.go

```diff
@@ -60,8 +60,12 @@ func (fp *FrontendPluginBase) handleModuleDefaults() {
 	}
 
 	fp.IsCorePlugin = true
-	fp.Module = path.Join("app/plugins", fp.Type, fp.Id, "module")
-	fp.BaseUrl = path.Join("public/app/plugins", fp.Type, fp.Id)
+	// Previously there was an assumption that the plugin directory
+	// should be public/app/plugins/<plugin type>/<plugin id>
+	// However this can be an issue if the plugin directory should be renamed to something else
+	_, currentDir := path.Split(fp.PluginDir)
```

Comment:
Nit. Could probably use https://golang.org/pkg/path/#Base instead.

Reply:
Note that a similar assumption is made that the plugin directory path will contain the plugin type (eg `datasource`). The plugin type is less probable to change but we could probably change that also. I was reluctant including it because this is something that affects all the frontend plugins.

Reply:
Should be fine as is